dynamiqs/utils/utils/__init__.py

```diff
@@ -27,4 +27,7 @@
     'fidelity',
     'eigenstates',
     'wigner',
+    'sincosm',
+    'sinm',
+    'cosm',
```

Comment:
I would probably put `sincosm` after `cosm` instead of before. Also, I would probably put these 3 functions after `mpow` because they have similar effects.